### PR TITLE
Add teacher standards

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDEL-504

### Is there anything weird that code reviewer should know?
Going commit by commit should be easiest.

I got annoyed by how often and how differently we handled govspeak content, so I made a component for it. And removed all other handling of govspeak content. 

### Review Checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests
- [x] All forms have an `id` attribute on them

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Sign in as admin
3. Go to any CIP lesson, click on "Edit lesson" button
4. Fill the ect teacher standards field, and the mentor teacher standards fiend (if you want)
5. Log out and log in as the ect / mentor that should be able to see that lesson
6. There should be an accordion with TS on the course module page (where you can see a list of course lessons)

### CODE REVIEW

File: app/controllers/core_induction_programmes/lessons_controller.rb [hunk 1]
```diff
@@ -62,6 +62,8 @@ def load_course_lesson
   end
 
   def course_lesson_params
-    params.require(:course_lesson).permit(:title, :mentor_title, :ect_summary, :mentor_summary, :completion_time_in_minutes, :course_module_id, :new_position)
+    params.require(:course_lesson).permit(
+      :title, :mentor_title, :ect_summary, :mentor_summary, :completion_time_in_minutes, :course_module_id, :new_position, :ect_teacher_standards, :mentor_teacher_standards
```
Comment (xswoodsx) [inline app/controllers/core_induction_programmes/lessons_controller.rb:65]: This better across multiple lines? Is there a neater way of dealing with so many params?

File: app/views/core_induction_programmes/modules/show.html.erb [hunk 2]
```diff
@@ -32,6 +32,21 @@
             <% end %>
           </h2>
 
+          <% if lesson.teacher_standards_for(current_user) %>
+            <details class="govuk-details  govuk-!-margin-top-2 govuk-!-margin-bottom-2" data-module="govuk-details">
+              <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                View which teachers' standards this topic relates to
+              </span>
+              </summary>
```
Comment (xswoodsx) [inline app/views/core_induction_programmes/modules/show.html.erb:40]: whitespace in details class 

Indenting on span? or does it sit like that? Feel like i've seen this before with span but is there something different about it's set like this?

File: db/migrate/20210607151925_add_teacher_standards_to_lessons.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddTeacherStandardsToLessons < ActiveRecord::Migration[6.1]
+  def change
+    change_table :course_lessons, bulk: true do |t|
+      t.string :ect_teacher_standards, null: true
+      t.string :mentor_teacher_standards, null: true
```
Comment (xswoodsx) [inline db/migrate/20210607151925_add_teacher_standards_to_lessons.rb:7]: The bulk true is combining altar table queries, what's the benefit in doing that? or is this something that always happens and i've just not noticed it?

Comment (kicferk1) [reply db/migrate/20210607151925_add_teacher_standards_to_lessons.rb:7]: Hehe, the benefit for me is shutting Rubucop up.

The slight benefit is that you don't need to specify the table name multiple times - `course_lessons` is just mentioned once.

**Review by xswoodsx — APPROVED** [review]
LGTM :)

**Pull request merged**